internal/build/tar.go

```diff
@@ -179,16 +189,18 @@ func (a *ArchiveBuilder) entriesForPath(ctx context.Context, source, dest string
 
 		if sourceIsDir {
 			// Name of file in tar should be relative to source directory...
-			header.Name = strings.TrimPrefix(path, source)
+			tmp, err := filepath.Rel(source, path)
```

Comment:
In Go, the `filepath` package is for the host OS paths, and the `path` package is for slash-separated paths paths. so maybe all this code should just use `path.Rel`, `path.Join`, `path.Separator`?

Reply:
@nicks  There is no such function path.Rel this is why i use filepath.Rel.